provider/cluster/kube/builder.go

```diff
@@ -287,9 +287,10 @@ func (b *deploymentBuilder) container() corev1.Container {
 // service
 type serviceBuilder struct {
 	deploymentBuilder
+	isGlobal bool
```

Comment:
I think that this is confusing because "global" doesn't necessarily mean that it should be a `NodePort` service.

Reply:
What is the intent of the "Global" flag then?

Reply:
Will refactor struct member name to `requiresNodePort`?